src/ui/control/geolocate_control.js

```diff
@@ -555,6 +614,9 @@ class GeolocateControl extends Evented {
     _clearWatch() {
         window.navigator.geolocation.clearWatch(this._geolocationWatchID);
 
+        window.removeEventListener('deviceorientationabsolute', this._onDeviceOrientation);
```

Comment:
Nit: we only do `addEventListener` for `deviceorientation`, should we add a corresponding `deviceorientationabsolute` too, or is this a leftover?

Reply:
@mourner Good catch! This is removed!